src/workflow/units/execution/vasp.py

```diff
@@ -12,3 +14,34 @@ class VaspExecutionUnit(BaseExecutionUnit):
 
     def __init__(self, config, work_dir):
         super(VaspExecutionUnit, self).__init__(config, work_dir)
+
+    @property
+    def stdout_file(self):
+        stdout_file = super(VaspExecutionUnit, self).stdout_file
+        return stdout_file if os.path.exists(stdout_file) else os.path.join(self.work_dir, "OUTCAR")
```

Comment:
1. https://cms.mpi.univie.ac.at/wiki/index.php/OSZICAR
2. I suggest using a variable and taking the name from settings

Reply:
1. `OSZICAR` only contains convergence data, but `OUTCAR` contains everything.
2. `OUTCAR` is used as default in here in case the stdout is not given. The stdout is given by user and it is parametric already.

Reply:
let's call it `VASP_DEFAULT_STDOUT` and put to settings. Stdout really goes to OSZICAR, that's what we show for VASP jobs on the platform. 

We can use OUTCAR, as it's more explanatory. It could be lengthy though.

Reply:
Done.